<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfazObra</class>
 <widget class="QWidget" name="InterfazObra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Med</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="botonAtras">
         <property name="text">
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonAdelante">
         <property name="text">
          <string>-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Ver</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboMedCert">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Medici贸n</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Certificaci贸n</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botonCopiar">
             <property name="text">
              <string>Copiar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botonPegar">
             <property name="text">
              <string>Pegar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botonGuardar">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="TablaPrincipal" name="tablaPrinc" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Medici贸n</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="TablaMedCert" name="TablaMed" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Certificaci贸n</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="TablaCert"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="Editor" name="editor" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Editor</class>
   <extends>QWidget</extends>
   <header>./Editor/editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TablaMedCert</class>
   <extends>QWidget</extends>
   <header>./Tablas/tablamedcert.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TablaPrincipal</class>
   <extends>QWidget</extends>
   <header>./Tablas/tablaprincipal.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
